### ISSUE TITLE
Can't log in

### ISSUE THREAD

**Issue by retro486** [Micahnator/Panpipe#6]
Login screen shows no errors or messages, just refreshes login screen.

**Comment by retro486** [comment]
On Nexus 4 OTA-15

**Comment by Micahnator** [comment]
Just to confirm, you are in the USA, Australia, or New Zealand, right? Panpipe blocks connections from any other country.

I run my N4 on the development channel. After seeing this I whipped it out and tried to run Panpipe and it wouldn't load. Would not even let me try to login. So I updated (likely a very minor update) from build 585 to build 587 and now it is running as I expect it to.

I'm guessing Panpipe was working for you before the update to OTA-15 (which just came out iirc)?

**Comment by retro486** [comment]
Yeah, I'm in the USA. I just double checked and I'm actually on OTA-14... But I could have sworn it was OTA-15 yesterday. So might be a 14 issue.

**Comment by Micahnator** [comment]
Looks like we were both off, OTA-14 was the most recent major update (http://www.omgubuntu.co.uk/2016/12/ubuntu-touch-ota-14-whats-new)

I might be able to flash OTA-14 on my device and check it out soon. I just did a fresh install of Ubuntu 16.10, so I need to reset up my tooling and so far I'm having some issues. I think the SDK has changed a bit since the last time I worked on Panpipe.

**Comment by retro486** [comment]
Ok figured it out. Pandora doesn't like my old username for authentication; now I have to use email address. So perhaps this bug request could instead be "change username field to an email address field". PR #7 submitted in relation to this.
